qa/src/test/java/io/camunda/migrator/qa/ValidationJobTypeOnlyTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+package io.camunda.migrator.qa;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.migrator.RuntimeMigrator;
+import io.camunda.migrator.qa.util.RuntimeMigrationAbstractTest;
+import io.github.netmikey.logunit.api.LogCapturer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = {
+    "camunda.migrator.validation-job-type==if legacyId then \"migrator\" else \"noop\""
+})
+public class ValidationJobTypeOnlyTest extends RuntimeMigrationAbstractTest {
```

Comment:
💭 Personally, because we already have the extensive config check that confirms when the validation is enabled/disabled and how the job type and validation job type properties are set, I feel like we dont need this class to confirm the behaviour when only the validation-job-type is set. However, it does add extra confidence - just thinking about how long our test suite already takes. Up to you